content/applications/sales/crm/performance/forecast_report.rst

```diff
@@ -0,0 +1,109 @@
+===============
+Forecast report
+===============
+
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |pivot| replace:: :icon:`oi-view-pivot` :guilabel:`(pivot)` icon
+.. |list| replace:: :icon:`oi-view-list` :guilabel:`(list)` icon
+
+The *Forecast* report in the *CRM* app allows users to view upcoming opportunities and build a
+forecast of potential sales. Opportunities are grouped by the month of their expected closing date,
+and can be dragged-and-dropped to adjust the deadline.
+
+To access the *Forecast* report, navigate to :menuselection:`CRM app --> Reporting --> Forecast`.
+
+.. image:: forecast_report/reporting-menu.png
+   :align: center
+   :alt: The location of the Forecast report in the CRM app.
+
+Navigate the forecast report
+============================
+
+The default forecast report includes opportunities assigned to the current user's pipeline, and
+expected to close within four months, as well as opportunities without an assigned expected closing
+date. The opportunities are grouped by month in a Kanban (:icon:`oi-view-kanban`) view.
+
+.. image:: forecast_report/sample-report.png
+   :align: center
+   :alt: A sample version of the Forecast report in the CRM app.
+
+Expected closing date
+---------------------
+
+Opportunities are grouped by the date assigned in the :guilabel:`Expected Closing` field. To change
+this date, select the Kanban card for the desired opportunity, then click and drag the card to the
+desired column.
+
+.. note::
+   The default time frame for the forecast is *month*. This can be changed by clicking the |caret|
+   next to the :guilabel:`Search...` bar at the top of the report. Under the :guilabel:`Group By`
+   heading, click :guilabel:`Expected Closing` to expand the list of available options, and select
+   from the list.
+
+After an opportunity is added to a new month, the :guilabel:`Expected Closing` field is updated to
+the *last* date of the new month.
+
+.. tip::
+   The :guilabel:`Expected Closing` field can also be manually updated. Click on the Kanban card for
+   an opportunity to open the opportunity's details form. Click in the :guilabel:`Expected Closing`
+   field, and use the calendar popover to select a new closing date.
+
+Prorated revenue
+----------------
+
+At the top of the column for each month, to the right of the progress bar, is a sum of the prorated
```

Comment:
```suggestion
At the top of the column for each month on the :guilabel:`Forecast` reporting page, to the right of the progress bar, is a sum of the prorated
```
OPTIONAL: just a helpful reminder to the reader, so they can always make sure they're on the right page